Doc/whatsnew/3.11.rst

```diff
@@ -329,14 +347,17 @@ See :pep:`675` for more details.
 (Contributed by Jelle Zijlstra in :issue:`47088`. PEP written by Pradeep
 Kumar Srinivasan and Graham Bleaney.)
 
+
+.. _whatsnew311-pep681:
+
 PEP 681: Data Class Transforms
 ------------------------------
 
 :data:`~typing.dataclass_transform` may be used to
 decorate a class, metaclass, or a function that is itself a decorator.
 The presence of ``@dataclass_transform()`` tells a static type checker that the
-decorated object performs runtime "magic" that
-transforms a class, giving it :func:`dataclasses.dataclass`-like behaviors.
+decorated object performs runtime "magic" that transforms a class,
+giving it :func:`dataclass <dataclasses.dataclass>`-like behaviors.
```

Comment:
```suggestion
giving it :func:`~dataclasses.dataclass`-like behaviors.
```

Reply:
Likewise I initially had this, but without a custom link text, it appends a `()` to the name, which is both syntactically disruptive and not semantically correct here, as it is referring to the objects decorated _by_ the dataclass decorator, not the dataclass function itself.

Reply:
Ah, fair enough!

Reply:
In other words, "great minds"... :)